pkgs/top-level/ocaml-packages.nix

```diff
@@ -412,6 +412,11 @@ let
       then callPackage ../development/ocaml-modules/ppx_deriving {}
       else null;
 
+    ppx_import =
+      if lib.versionAtLeast ocaml.version "4.02"
```

Comment:
I'm already giving `minimumSupportedOcamlVersion` in the derivation. Does it means that I can just drop the conditional?